crud/src/test/java/com/redhat/lightblue/mediator/MediatorTest.java

```diff
@@ -401,6 +400,72 @@ public void queryTimeoutTest() throws Exception {
     }
 
     @Test
+    public void findResultSetTooLargeTest() throws Exception {
```

Comment:
Really simple amongst not simple code :-). Simple things break all the time, that's what simple tests are for. Anyway, it's not a huge deal I guess.